frontend/public/components/utils/firehose.jsx

```diff
@@ -174,7 +174,7 @@ export const Firehose = connect(
         if (expand !== this.props.expand) {
           return true;
         }
-        return false;
+        return nextProps.forceUpdate;
```

Comment:
Why is this needed?

Reply:
Basically, Firehose will only re-render when the underlying resources change. So switching the view component rendered inside the Firehose will not cause a re-render. Since we are taking out the view switching functionality, this probably isn't necessary right now. We could also probably figure out a better way to do this. I just hacked something together to work temporarily.